src/common/common.go

```diff
@@ -42,3 +44,20 @@ func SetConfiguringStatusForHosts(client inventory_client.InventoryClient, inven
 		}
 	}
 }
+
+func IsPodInStatus(k8Client k8s_client.K8SClient, podNamePrefix string, namespace string, labelMatch map[string]string, status v1.PodPhase, log logrus.FieldLogger) bool {
+	pods, err := k8Client.GetPods(namespace, labelMatch)
+	if err != nil {
+		log.WithError(err).Warnf("Failed to get pod with prefix %s in namespace %s", podNamePrefix, namespace)
+		return false
+	}
+	for _, pod := range pods {
```

Comment:
You expect more than 1?

Reply:
it is common , yes it is possible for sure

Reply:
OK, consider logging all the pods you find with this selector just so we will see what is going on.

Reply:
didn't wanted to spawn too much logs